src/prefect/client/orion.py

```diff
@@ -199,6 +200,8 @@ def __init__(
 
         self.logger = get_logger("client")
 
+        self.httpx_settings = httpx_settings
```

Comment:
Is there a better way to retrieve the `verify` setting?

Reply:
Yeah I would not do it this way. I'd mock the `httpx.Client` and assert that it is called with the correct value.